rosetta/common/config.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	// RosettaVersion tied back to the version of the rosetta go-sdk
-	RosettaVersion = "0.4.4" // TODO (dm): set variable via build flags
+	RosettaVersion = "1.4.6" // TODO (dm): set variable via build flags
```

Comment:
1.4.6 or 0.4.6?

Reply:
1.x.x since it's now a release. I previously set it to 0.x.x since we were still working on it.

I've made it match https://www.rosetta-api.org/versions.html.